### PR TITLE
Docs for E20 custom terms of service

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/custom-terms-of-service.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Custom Terms of Service (Beta) (E20)
+=====================================
+
+*Available in `Enterprise Edition E20 and higher <https://about.mattermost.com/pricing/>`_.*
+
+In Mattermost Enterprise Edition E20, you can outline custom Terms of Service for your team members to accept before they can access Mattermost on web or desktop.
+
+.. note::
+
+ Users on mobile do not have to accept the terms of service with mobile support scheduled for an upcoming release.
```
Comment (wiersgallak) [inline source/administration/custom-terms-of-service.rst]: Consider revising "Users on mobile will not be presented with the custom Terms of Services.  Mobile support is scheduled for an upcoming release."  '...do not have to accept' may imply that the functionality is different, when it is not available yet.

File: source/administration/custom-terms-of-service.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Configuring Terms of Service
+--------------------------------
+
+To enable custom terms of service:
+
+1. Go to **System Console > Customization > Legal and Support**.
+2. Set **Enable Custom Terms of Service** to true.
```
Comment (wiersgallak) [inline source/administration/custom-terms-of-service.rst]: Does this need formatting?

File: source/administration/custom-terms-of-service.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+There will be no impact to your end users. Users will simply be required to accept the new terms on next login or page refresh.
+
+How do I know if my users accepted the terms of service?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each time a user agrees to the terms of service, the agreement is recorded in the database:
+
+ - A new field <XXX> in the **Users** table indicates if a user has accepted the most recent terms of service.
+ - Each time a user accepts the terms, the action is recorded in the **Audits** table, containing the username, timestampt of acceptance, and other relevant information.
```
Comment (wiersgallak) [inline source/administration/custom-terms-of-service.rst]: spelling - timestamp

File: source/administration/custom-terms-of-service.rst (hunk 1 above)
Comment (jasonblais) [reply source/administration/custom-terms-of-service.rst]: Good piece of feedback, updated

**Comment by jasonblais** [comment]
@wiersgallak Ready for re-review 👍

File: source/administration/custom-terms-of-service.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Frequently Asked Questions
+----------------------------
+
+What happens if I update my terms of service text?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There will be no impact to your end users. Users will simply be required to accept the new terms on next login or page refresh.
+
+How do I know if my users accepted the terms of service?
```
[the remark this answers is not in the document]
Comment (grundleborg) [reply source/administration/custom-terms-of-service.rst]: I don't really know. The answer is "look in the database / audit history" I think.

File: source/administration/custom-terms-of-service.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each time a user agrees to the terms of service, the agreement is recorded in the database:
+
+ - A new field <XXX> in the **Users** table indicates if a user has accepted the most recent terms of service.
```
[the remark this answers is not in the document]
Comment (grundleborg) [reply source/administration/custom-terms-of-service.rst]: `AcceptedServiceTermsId` is the only new field in the Users table, which contains the Id of the most recent TOS that user accepted.

**Comment by jasonblais** [comment]
Thanks George, updated based on your feedback

**Pull request merged**